tests/integration/requirements.txt

```diff
@@ -0,0 +1,137 @@
+appdirs==1.4.3
```

Comment:
i would prefer we have a requirements file and a constraints file. we should also be using hashes.